templates/web/base/waste/bulky/summary.html

```diff
@@ -1,28 +1,157 @@
+        <small >([% remaining %] remaining slot[% 's' IF remaining > 1 %] available)</small>
+      </div>
+
+      <table class="item-summary-table govuk-!-margin-bottom-9">
+        <thead>
+          <tr>
+              <th>Units</th>
+              <th>Item</th>
+              <th>
+                <span class="govuk-text__assistive">Item images preview</span>
+              </th>
+          </tr>
+        </thead>
+        <tbody>
+          [% FOR item IN items %]
+          [% extra_text = form.items_extra_text.${item.item} %]
+          <tr [% IF extra_text %]class="is--no-border-bottom"[% END %]>
+              <td class="item-number-cell">1</td> [% #XXX would be nicer to group together, but photos? %]
+              <td>
+                <p class="govuk-!-margin-bottom-0">[% item.item %]</p>
+              </td>
+              <td>
+                  [% IF item.photo_fileid %]
+                    <img class="img-preview is--small" alt="Preview image succesfully attached" src="/photo/temp.[% item.photo_fileid %]">
+                  [% ELSE %]
+                    <span class="govuk-text__assistive">No image attached for this item</span>
+                  [% END %]
+              </td>
+          </tr>
+          [% IF extra_text %]
+            <tr>
+              <td colspan="3">
+                <div class="govuk-warning-text due" style="padding:1em">
+                  <div class="govuk-warning-text__img">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                  </div>
+                  <div class="govuk-warning-text__content" style="display:flex; align-items:center;">
+                      <span class="govuk-warning-text__assistive">Warning for item: [% item.item %]</span>
+                      <p class="govuk-!-margin-bottom-0">[% extra_text %]</p>
+                  </div>
+                </div>
+              </td>
+            </tr>
           [% END %]
-        </dd>
+        [% END %]
+        </tbody>
+      </table>
+
+
+      <h3>Location details</h3>
+      <p>
+        [% IF data.location %]
+          [% data.location %]
+        [% ELSE %]
+          No description provided.
+        [% END %]
+      </p>
+      [% IF data.location_photo_fileid %]
+        <img class="img-preview is--medium" alt="Preview image succesfully attached" src="/photo/temp.[% data.location_photo_fileid %]">
+      [% ELSE %]
+        <p class="due" style="padding:1em">No image of the location has been attached.</p>
+      [% END %]
     </div>
-  [% END %]
-[% END %]
 
-[% PROCESS waste/summary.html %]
+    <div class="govuk-grid-column-one-third">
+      <h3>Your details</h3>
+      <dl>
+        <dt>Name</dt>
+        <dd>[% data.name %]</dd>
+        <dt>Email</dt>
+        <dd>[% data.email %]</dd>
+        <dt>Phone</dt>
+        <dd>[% data.phone OR 'Not given' %]</dd>
+      </dl>
+
+      [% IF data.on_behalf_of # XXX %]
```

Comment:
I don't think this section is needed - if you're entering on behalf of someone else, that'll be the name/email/etc given above, there's no separate "your details" (guess could show them if logged in staff, but doesn't seem necessary).